src/core/common.hpp

```diff
@@ -70,6 +73,21 @@
     #define forceinline __attribute__(( always_inline )) inline
 #endif
 
+namespace rapidgzip
```

Comment:
This would be more apt inside `src/core/TestHelpers.hpp` because it is only intended for debugging output and does not define any stable output format.